csrankings-a.csv

```diff
@@ -2480,6 +2481,7 @@ Aws Albarghouthi,University of Wisconsin - Madison,http://pages.cs.wisc.edu/~aws
 Axel Hahn,University of Oldenburg,https://uol.de/sao/personen/prof-dr-ing-axel-hahn,0XGwMHkAAAAJ
 Axel Jantsch,TU Wien,http://jantsch.se/AxelJantsch,ZF4gC44AAAAJ
 Axel Kalenborn,University of Trier,https://www.uni-trier.de/index.php?id=17390,NOSCHOLARPAGE
+Axel Krieger,Johns Hopkins University,https://me.jhu.edu/faculty/axel-krieger/,ezJ0eHAAAAAJ

```

Comment:
<img width="1225" alt="Screen Shot 2024-11-04 at 10 42 17 AM" src="https://github.com/user-attachments/assets/5782625d-9578-4ae9-8edd-9cdad3533b27">

https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/